### PR TITLE
Change VTX power level via switch

### PR DESCRIPTION
**Pull request by L4ky**
Hi, 
this PR allow user to change VTX power during the flight.
I used "Adjustments" mechanism to avoid huge modification to iNav's code.

I'm not sure in "rc_adjustments.c" if it's correct to use "VTX_SETTINGS_MIN_POWER" and "VTX_SETTINGS_MAX_POWER" to constrain the variable which sets the power level.

My advice is to use a 3-pos switch or a potentiometer and set the same channel in "Range Channel" and "Adjustment Channel".

I tested it only on my Eachine Smartaudio VTX.

Note: i have PR ready for inav-configurator to add this feature. I'll wait for this to get accepted before proposing that.

EDIT: iNav Configurator PR [#775](https://github.com/iNavFlight/inav-configurator/pull/775)

### CODE REVIEW

**Comment by L4ky** [comment]
While i wait for approval or changes, iNav Configurator PR: [#775](https://github.com/iNavFlight/inav-configurator/pull/775)

**Review by shellixyz — CHANGES_REQUESTED** [review]
Please rebase your branch upon the development branch and fix the indentation. We are using 4 spaces per level and no tabs.

Otherwise looking good to me

**Comment by L4ky** [comment]
Thank you @shellixyz 

After some mess i should have rebased correctly.

**Review by shellixyz — APPROVED** [review]
Since you originally based your branch upon master it picked up 3747ca451 but it is not a big deal. LGTM

**Comment by whoim2** [comment]
This working now? (Change vtx power via swith)

**Pull request merged**
